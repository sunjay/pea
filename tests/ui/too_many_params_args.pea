fn main() {
    foo(
        param0,
        param1,
        param2,
        param3,
        param4,
        param5,
        param6,
        param7,
        param8,
        param9,
        param10,
        param11,
        param12,
        param13,
        param14,
        param15,
        param16,
        param17,
        param18,
        param19,
        param20,
        param21,
        param22,
        param23,
        param24,
        param25,
        param26,
        param27,
        param28,
        param29,
        param30,
        param31,
        param32,
        param33,
        param34,
        param35,
        param36,
        param37,
        param38,
        param39,
        param40,
        param41,
        param42,
        param43,
        param44,
        param45,
        param46,
        param47,
        param48,
        param49,
        param50,
        param51,
        param52,
        param53,
        param54,
        param55,
        param56,
        param57,
        param58,
        param59,
        param60,
        param61,
        param62,
        param63,
        param64,
        param65,
        param66,
        param67,
        param68,
        param69,
        param70,
        param71,
        param72,
        param73,
        param74,
        param75,
        param76,
        param77,
        param78,
        param79,
        param80,
        param81,
        param82,
        param83,
        param84,
        param85,
        param86,
        param87,
        param88,
        param89,
        param90,
        param91,
        param92,
        param93,
        param94,
        param95,
        param96,
        param97,
        param98,
        param99,
        param100,
        param101,
        param102,
        param103,
        param104,
        param105,
        param106,
        param107,
        param108,
        param109,
        param110,
        param111,
        param112,
        param113,
        param114,
        param115,
        param116,
        param117,
        param118,
        param119,
        param120,
        param121,
        param122,
        param123,
        param124,
        param125,
        param126,
        param127,
        param128,
        param129,
        param130,
    );
}

fn foo(
    param0: i64,
    param1: i64,
    param2: i64,
    param3: i64,
    param4: i64,
    param5: i64,
    param6: i64,
    param7: i64,
    param8: i64,
    param9: i64,
    param10: i64,
    param11: i64,
    param12: i64,
    param13: i64,
    param14: i64,
    param15: i64,
    param16: i64,
    param17: i64,
    param18: i64,
    param19: i64,
    param20: i64,
    param21: i64,
    param22: i64,
    param23: i64,
    param24: i64,
    param25: i64,
    param26: i64,
    param27: i64,
    param28: i64,
    param29: i64,
    param30: i64,
    param31: i64,
    param32: i64,
    param33: i64,
    param34: i64,
    param35: i64,
    param36: i64,
    param37: i64,
    param38: i64,
    param39: i64,
    param40: i64,
    param41: i64,
    param42: i64,
    param43: i64,
    param44: i64,
    param45: i64,
    param46: i64,
    param47: i64,
    param48: i64,
    param49: i64,
    param50: i64,
    param51: i64,
    param52: i64,
    param53: i64,
    param54: i64,
    param55: i64,
    param56: i64,
    param57: i64,
    param58: i64,
    param59: i64,
    param60: i64,
    param61: i64,
    param62: i64,
    param63: i64,
    param64: i64,
    param65: i64,
    param66: i64,
    param67: i64,
    param68: i64,
    param69: i64,
    param70: i64,
    param71: i64,
    param72: i64,
    param73: i64,
    param74: i64,
    param75: i64,
    param76: i64,
    param77: i64,
    param78: i64,
    param79: i64,
    param80: i64,
    param81: i64,
    param82: i64,
    param83: i64,
    param84: i64,
    param85: i64,
    param86: i64,
    param87: i64,
    param88: i64,
    param89: i64,
    param90: i64,
    param91: i64,
    param92: i64,
    param93: i64,
    param94: i64,
    param95: i64,
    param96: i64,
    param97: i64,
    param98: i64,
    param99: i64,
    param100: i64,
    param101: i64,
    param102: i64,
    param103: i64,
    param104: i64,
    param105: i64,
    param106: i64,
    param107: i64,
    param108: i64,
    param109: i64,
    param110: i64,
    param111: i64,
    param112: i64,
    param113: i64,
    param114: i64,
    param115: i64,
    param116: i64,
    param117: i64,
    param118: i64,
    param119: i64,
    param120: i64,
    param121: i64,
    param122: i64,
    param123: i64,
    param124: i64,
    param125: i64,
    param126: i64,
    param127: i64,
    param128: i64,
    param129: i64,
    param130: i64,
) {
    println!(b"foo");
}
