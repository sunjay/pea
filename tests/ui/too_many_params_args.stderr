[tests/ui/too_many_params_args.pea:3:9-133:16] error: function calls may have up to 127 arguments but 131 were provided

[tests/ui/too_many_params_args.pea:138:5-268:17] error: functions may have up to 128 parameters but 131 were provided

error: aborting due to 2 previous errors

